components/Avatar/style.css

```diff
@@ -0,0 +1,50 @@
+.avatar{
+    border-radius:50%;
+  }
```

Comment:
A suggestion here would be that the indentation looks a bit off here and the consecutive lines. Maybe you could correct them.

Reply:
okay i resolved that